packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -227,4 +239,5 @@ export enum FetchType {
     snaphsotTree = "snapshotTree",
     treesLatest = "treesLatest",
     uploadSummary = "uploadSummary",
+    push = "push",
```

Comment:
Consider removing enum and either just using strings, or defining union type, i.e.
type FetchType = "blob" | ...;

I like enums and they are the right mechanism for many things, but they do add cost in bundles, so for places where the only purpose is to add a string to telemetry, I think it's fine to either just use strings (and not assert that it needs to be one from pre-defined list) or use union type.